web/src/components/GameMaker.jsx

```diff
@@ -294,8 +301,24 @@ export default function GameMaker({game, newSave, editSave, gamemakerIndex}) {
 
                 <Grid container item xs={2}></Grid>
             </Grid>
-        </form>
-    );
+
+            <Grid container xs={2}></Grid>
```

Comment:
I think this was accidentally repeated so this line can be removed since it is already in line 302